docs/installation/Setup-OpenFunction-Builder-with-Tekton.md

```diff
@@ -0,0 +1,154 @@
+# Setup OpenFunction Builder with Tekton
+
+You can refer to the installation steps on the [Tekon docs](https://tekton.dev/docs/getting-started/) for setup OpenFunction Builder. Or follow these steps:
+> :grey_exclamation: Refer to [this section](#Installation with poor network connections to GitHub/Googleapis) when you are in a **poor network connections to GitHub/Googleapis**.
+
+## Installation
+
+### Install tekton pipeline
+
+```bash
+kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
+```
+
+#### Confirm that every component listed has the status Running
+
+```bash
+kubectl get pods --namespace tekton-pipelines
+
+NAME                                          READY   STATUS    RESTARTS   AGE
+tekton-pipelines-controller-6b94f5f96-hdjw5   1/1     Running   0          20m
+tekton-pipelines-webhook-5bfbbd6475-6fl2r     1/1     Running   0          20m
+```
+
+### Install tekton CLI
+
+Choose a suitable installation of tekton CLI for your cluster by refer to thie [docs](https://tekton.dev/docs/cli/).
+
+### Install tekton triggers
+
+```bash
+kubectl apply --filename https://storage.googleapis.com/tekton-releases/triggers/latest/release.yaml
+```
+
+### Install tekton Dashboard
+
+```bash
+kubectl apply --filename https://github.com/tektoncd/dashboard/releases/latest/download/tekton-dashboard-release.yaml
+```
+
+If you want to use ```NodePort``` to expose the tekton dashboard service, you need to modify the ```spec``` fields of ```svc tekton-dashboard``` like below：
+
+1. Use this command:
+    ```bash
+    kubectl edit -n tekton-pipelines svc tekton-dashboard
+    ```
+2. And modify the information:
+    ```yaml
+    spec:
+      clusterIP: 10.233.56.213
+      externalTrafficPolicy: Cluster
+      ports:
+      - name: http
+        nodePort: 31003 # Select a suitable port
+        port: 9097
+        protocol: TCP
+        targetPort: 9097
+      selector:
+        app.kubernetes.io/component: dashboard
+        app.kubernetes.io/instance: default
+        app.kubernetes.io/name: dashboard
+        app.kubernetes.io/part-of: tekton-dashboard
+      sessionAffinity: None
+      type: NodePort # change to NodePort
+    ```
+
+### Integration with Cloud Native Buildpacks
```

Comment:
Cloud Native Buildpacks are automatically created by OF code, needn't to install git-clone and buildpack tasks